homeassistant/components/zwave_js/__init__.py

```diff
@@ -252,6 +261,11 @@ def async_on_notification(notification: Notification) -> None:
     try:
         async with timeout(CONNECT_TIMEOUT):
             await client.connect()
+    except InvalidServerVersion as err:
+        LOGGER.error("Invalid server version: %s", err)
+        if use_addon:
+            async_ensure_addon_updated(hass)
```

Comment:
Shouldn't we await this?

Reply:
I was thinking we don't want to hold up the set up phase.